### PR TITLE
Yuzu android guide is changed to letest yuzu android version guide

### PR DESCRIPTION
**Pull request by Ishan0981**

### CODE REVIEW

File: docs/yuzu_android.md [hunk 1]
```diff
@@ -4,18 +4,18 @@ Tap on the Settings cog on the main screen of the app:
 
 <p align="center"><img src="images/yuzu_settings.png"/></p>
 
-Tap on "Install GPU driver":
+Tap on "GPU driver manager":
 
-<p align="center"><img src="images/yuzu_gpu.png"/></p>
+<p align="center"><img src="https://github.com/K11MCH1/AdrenoToolsDrivers/assets/145665569/cf8d42c6-b5bf-47f7-a002-dc5b2cd6574a"/></p>
```
Comment (Crytonics) [inline docs/yuzu_android.md:9]: Again use image files, not links

File: docs/yuzu_android.md [hunk 2]
```diff
@@ -4,18 +4,18 @@ Tap on the Settings cog on the main screen of the app:
...
 
 Tap on "Install":
 
-<p align="center"><img src="images/yuzu_install.png"/></p>
+<p align="center"><img src="https://github.com/K11MCH1/AdrenoToolsDrivers/assets/145665569/bd87c779-1fc7-4c0f-98db-82aec4ba619f"/></p>
```
Comment (Crytonics) [inline docs/yuzu_android.md:13]: Again use image files, not links

**Review by Crytonics — CHANGES_REQUESTED** [review]

**Comment by Crytonics** [comment]
It is for better maintaning and easier to read. Also when they need to update image they can just upload image file with same name, no need to change code

**Comment by Ishan0981** [comment]
I will change don't worry

**Pull request merged**